src/frontend/src/pages/ChangeRequestsPage/ChangeRequestsTable.tsx

```diff
@@ -120,12 +122,14 @@ const ChangeRequestsTable: React.FC = () => {
         title={'Change Requests'}
         previousPages={[]}
         actionButton={
-          <ActionButton
-            link={routes.CHANGE_REQUESTS_NEW}
-            icon={faPlus}
-            text={'New Change Request'}
-            disabled={auth.user?.role === 'GUEST'}
-          />
+          <Link className={'row py-auto px-3 '} to={routes.CHANGE_REQUESTS_NEW} style={{ textDecoration: 'none' }}>
+            <Button variant="text">
+              New Change Request
+              <IconButton onClick={() => (window.location.href = routes.CHANGE_REQUESTS_NEW)}>
```

Comment:
Having a separate IconButton component will lead us to have two button (one for whole 'new change request' button and another for only '+' sign) If the users clicks the '+' sign, it will redirect them to login page, and the user have to login again to access new change request form page. Therefore, I recommend putting <Add /> mui icon component in Button component on line 126 as an **endIcon** field.